### PR TITLE
Add blockgen command to generate blocks of mock data

### PR DESCRIPTION
**Pull request by replay**

### CODE REVIEW

**Review by bboreham — COMMENTED** [review]
Nice!

However there is a compile error - is something missing?

**Comment by replay** [comment]
> Nice!
> 
> However there is a compile error - is something missing?

Hey, thanks for checking it out!
Sorry about that, I forgot to commit a change in a test file. 
The CI is passing now

File: pkg/commands/block_gen.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	seriesSet, totalSeriesTypeMap := bench.SeriesDescToSeries(f.Series)
+	totalSeries := 0
+	for _, typeTotal := range totalSeriesTypeMap {
+		totalSeries += typeTotal
+	}
+
+	writeWorkLoad := bench.WriteWorkload{
+		TotalSeries:        totalSeries,
+		TotalSeriesTypeMap: totalSeriesTypeMap,
+		Replicas:           1,
```
Comment (jtlisi) [inline pkg/commands/block_gen.go:72]: Maybe worth making this configurable? It's useful as a lever to configure the number of metrics total quickly. For example, create a workload w/ 50k metrics per replica. You can just raise the replica to have 100k, 500k, etc...

```suggestion
		Replicas:           1,
```

**Review by jtlisi — APPROVED** [review]
LGTM

File: pkg/commands/block_gen.go (hunk 1 above)
Comment (replay) [reply pkg/commands/block_gen.go]: Good idea, updated that: https://github.com/grafana/cortex-tools/pull/191/commits/ddeff2954cb36d628c94c6b53708fe28b7b6a8b5

**Pull request merged**